### DISCUSSION TITLE
Mask didn't trigger on backspace

### DISCUSSION THREAD

**Discussion by douxlagithub** [alpinejs/alpine#4295] [7. Bugs (archived)]
The mask didn't trigger changes on the dot position when removing inputs using backspace. It is nice to see the realtime changing just like when we type in the input, the dots position is changes as we type. Not happen on backspace.

**Comment by ekwoka** [comment]
This is related to the current implementation that eagerly injects template characters. Without it, those would be harder to delete

However, a rework on the implementation is open here #4260 so you can read the discussion and give some input.
